modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

```diff
@@ -123,15 +123,15 @@ To help you review your code, learn more about the different changes in Java 17:
 
 * JDK Migration Guide
 +
-See the https://docs.oracle.com/en/java/javase/17/migrate/getting-started.html#GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle JDK Migration Guide] for a full list of the changes made in Java 17. Use the guide to assess the impact to your code.
+See the https://docs.oracle.com/en/java/javase/17/migrate/getting-started.html#GUID-C25E2B1D-6C24-4403-8540-CFEA875B994A[Oracle JDK Migration Guide] for a full list of the changes made from Java 8 to Java 17. Use the guide to assess the impact to your code.
 
 * Mule runtime engine
 +
 See xref:general::java-support.adoc#mule-runtime[Mule runtime] to learn more about the Java 17 changes made in Mule runtime engine. 
 +
 ** Reflection on Java classes
 +
-To ensure compatibility with Java 17, review and update any code that uses reflection to access Java classes or set parameter values. Java 17 imposes stricter access controls, so you should avoid introspecting JDK internals. Ensure you have a robust test suite and make sure it is run with Java 17.
+To ensure compatibility with Java 17, review and update any code that uses reflection to access Java classes or set parameter values. Java 17 imposes stricter access controls, so you should avoid introspecting JDK internals. Ensure you have a robust test suite and make sure it is run with Java 17. Pay special attention to the `setAccessible(true)` method because these type of changes in the accessibility is not allowed in Java 17.
```

Comment:
```suggestion
To ensure compatibility with Java 17, review and update any code that uses reflection to access Java classes or set parameter values. Java 17 imposes stricter access controls, so you should avoid introspecting JDK internals. Ensure you have a robust test suite and make sure it is run with Java 17. Pay special attention to the `setAccessible(true)` method because these types of changes in the accessibility is not allowed in Java 17.
```

Reply:
"this type of change" or even "these types of changes" suggests that there could be other accessibility changes not mentioned here. so I'm wondering what those could be. 

Maybe this means "because this accessibility change is not allowed in Java 17" or something ¯\_(ツ)_/¯